packages/cozy-stack-client/src/AppCollection.js

```diff
@@ -60,7 +60,8 @@ class AppCollection extends DocumentCollection {
           this.stackClient,
           `${this.endpoint}${encodeURIComponent(id)}`,
           {
-            normalize: data => normalizeApp(data, this.doctype)
+            normalize: data => normalizeApp(data, this.doctype),
+            dataForNotFound: null
```

Comment:
Any specific reason to add this option?

Reply:
At first, I wanted to return an empty array for data as the same we do when we `/_find` nothing. But after that, I saw that `Collection.get()` returned a `null` result. 

In fact, I still don't be confortable with the PR since : 
- `FilesCollection.get()` use `statById` which call directly `/files/${id}`. This route returns the same errors as /apps or /konnectors. So it should still throw after this PR. But there is no test that check that behavior. 
- I don't know if we should throw or not in fact in that case instead of returning a null value. I think that it can depends on the context : 
   - If I know I should have the file or the app, if a not found is raised, I should throw. 
   - If I know that it a possibility that I don't have the app installed, then I should not throw?